ui/src/app/shared/shared.module.ts

```diff
@@ -133,7 +135,8 @@ export function SubnetmaskValidatorMessage(err, field: FormlyFieldConfig) {
     ChartOptionsComponent,
     HeaderComponent,
     HistoryDataErrorComponent,
-    PercentageBarComponent
+    PercentageBarComponent,
+    SkeletonWrapperComponent
```

Comment:
i think FormlyFieldWithLoadingAnimation would be better, a name to describe whats happening